### PR TITLE
fixing objsorter problems and update ref key

### PR DESCRIPTION
**Pull request by felipeaoli**

### CODE REVIEW

File: backend/framework/qlf/dashboard/bokeh/qaskypeak/main.py [hunk 1]
```diff
@@ -70,6 +72,7 @@ def load_qa(self):
         c1, c2 = 0, 500
         qlf_fiberid = np.arange(0, 500)  # [c1:c2]
 
+        with open('dbg','w') as f: f.write(str(gen_info.keys()))
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qaskypeak/main.py]: Is this needed?

File: backend/framework/qlf/dashboard/bokeh/qaskypeak/main.py [hunk 2]
```diff
@@ -32,6 +32,8 @@ def load_qa(self):
         par = mergedqa['TASKS']['CHECK_SPECTRA']['PARAMS']
 
         gen_info = mergedqa['GENERAL_INFO']
+        import json
+        with open('dbg2.json','w') as f: json.dump(mergedqa, f)
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qaskypeak/main.py]: Is this needed?

**Comment by felipelm** [comment]
@felipeaoli sorry I thought it was up for revision already.

**Comment by felipeaoli** [comment]
Sorry, I'm still checking the other two plots..

File: backend/framework/qlf/dashboard/bokeh/qlf_plot.py [hunk 3]
```diff
@@ -242,12 +242,19 @@ def mtable(qa, data, objtype=['OTYPE ?', 'OTYPE ?']):
             program_prefix = '_'+program
         else:
             program_prefix = ''
-        ref=par[key+program_prefix+'_REF']
-        if isinstance(ref,list):
+        try:
+            ref=par[key+program_prefix+'_REF']
+        except:
+            if qa =='xyshifts':
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qlf_plot.py]: seems like we will remove this code pretty soon

File: backend/framework/qlf/dashboard/bokeh/qlf_plot.py [hunk 4]
```diff
@@ -293,7 +303,9 @@ def mtable(qa, data, objtype=['OTYPE ?', 'OTYPE ?']):
         # per_TGT
         if objtype is not None:
             objtype_tb = ['STAR' if i == 'STD' else i for i in objtype]
-            key_tb = [qa_metrics[qa]] * nrows
+            key_tb = [qa_metrics[qa]+" (%s)"%x for x in objtype_tb ]# * nrows
+            # key_tb = [qa_metrics[qa] + ' ( %s)' % objtype_tb[i]
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qlf_plot.py]: I prefer to remove this unused code

File: backend/framework/qlf/dashboard/bokeh/qasnr/main.py [hunk 5]
```diff
@@ -38,9 +38,15 @@ def load_qa(self):
         exposure = process.exposure
         fmap = Fibermap.objects.filter(exposure=exposure)
 
-        otype_tile = fmap[0].objtype
+        #otype_tile = fmap[0].objtype
 
-        objlist = sorted(set(otype_tile))
+        # Load data from Sframe fits:
+        #rame = load_fits(self.selected_process_id, cam)
+        #objlist = set(frame["FIBERMAP"].data["OBJTYPE"])
```
Comment (felipelm) [inline backend/framework/qlf/dashboard/bokeh/qasnr/main.py]: I would remove this as well

File: backend/framework/qlf/dashboard/bokeh/qlf_plot.py (hunk 3 above)
Comment (felipeaoli) [reply backend/framework/qlf/dashboard/bokeh/qlf_plot.py]: I believe so..

**Pull request merged**